### PR TITLE
Summarize ingress and egress at the top in Confluent Cloud dashboard

### PR DESCRIPTION
**Pull request by tsuz**
| Item | Changes |
|--|--|
| Write on Topics | Change from Graph widget to Stats (since graph exists below and total ingress/sec doesn't exist) |
| Read on Topics | Change from Graph widget to Stats (since graph exists below and total egress/sec doesn't exist)  |
| Request (rate) | Change title to `Request Count (rate)` from `Request (rate)` |

**Before**
Screenshot 2023-09-27 at 11 13 24

**After**

Screenshot 2023-09-27 at 11 27 53

### CODE REVIEW

File: ccloud-prometheus-grafana/assets/grafana/provisioning/dashboards/ccloud.json [hunk 1]
```diff
@@ -2484,5 +2387,6 @@
   "timezone": "",
   "title": "Confluent Cloud",
   "uid": "fhX5408Zk",
-  "version": 6
-}
\ No newline at end of file
+  "version": 9,
+  "weekStart": ""
```
Comment (hifly81) [inline ccloud-prometheus-grafana/assets/grafana/provisioning/dashboards/ccloud.json:2391]: @tsuz is "weekStart" key required here?

**Review by OoRobFuoO — CHANGES_REQUESTED** [review]
Please verify and confirm the grafana dashboard will also work for Grafana 8.1?

File: ccloud-prometheus-grafana/assets/grafana/provisioning/dashboards/ccloud.json [hunk 2]
```diff
@@ -75,21 +84,20 @@
...
+          ],
           "fields": "",
           "values": false
         },
         "showThresholdLabels": false,
         "showThresholdMarkers": true,
         "text": {}
       },
-      "pluginVersion": "8.1.3",
+      "pluginVersion": "10.1.2",
```
[the remark this answers is not in the document]
Comment (tsuz) [reply ccloud-prometheus-grafana/assets/grafana/provisioning/dashboards/ccloud.json]: Reverted back to 8.1.3. Tested with that version also, thanks.

File: ccloud-prometheus-grafana/assets/grafana/provisioning/dashboards/ccloud.json (hunk 1 above)
Comment (tsuz) [reply ccloud-prometheus-grafana/assets/grafana/provisioning/dashboards/ccloud.json]: Removed it.

**Review by hifly81 — APPROVED** [review]
LGTM!

**Comment by hifly81** [comment]
@tsuz any ideas why license/cla is missing?

**Comment by tsuz** [comment]
@hifly81 Ok, fixed the CLA issue. My git command line had different email config to my primary email.

**Comment by hifly81** [comment]
> @hifly81 Ok, fixed the CLA issue. My git command line had different email config to my primary email.

waiting the approval by
@OoRobFuoO

**Pull request merged**